examples/pylab_examples/mri_with_eeg.py

```diff
@@ -1,77 +1,79 @@
-"""
-This now uses the imshow command instead of pcolor which *is much
-faster*
+"""Displays a set of subplots with an MRI image, its intensity histogram and
+some EEG traces.
+
+NB: ones uses the imshow command instead of pcolor which *is much faster*.
 """
 
 from __future__ import division, print_function
 
 import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.cbook as cbook
+import matplotlib.cm as cm
 
-from matplotlib.pyplot import *
 from matplotlib.collections import LineCollection
-import matplotlib.cbook as cbook
-# I use if 1 to break up the different regions of code visually
+from matplotlib.ticker import MultipleLocator
 
-if 1:   # load the data
-    # data are 256x256 16 bit integers
+# NB: one uses "if 1:" to break up the different regions of code visually
+fig = plt.figure("MRI_with_EEG")
+
+if 1:   # Load the data
```

Comment:
I really dislike this syntay. Can we get rid of this? It is using python code for documentation purposes.
Having comments should be enough to break the different regions of the code visually.